testing/ncids-css-testing/src/stories/templates/blog-posts/blog-content.js

```diff
@@ -0,0 +1,149 @@
+import { leadImage } from './blog-lead-image';
+import { doctorAEmbed, doctorBEmbed } from './blog-images';
+import footer from './blog-footer';
+
+const html = `
+	<main id="main-content" class="contentzone">
```

Comment:
I am iffy on `<main>` being here or in the main template. It should probably go into the thing you are using for the 2 columns, wrapping the left column where this is placed.